app/views/profiles/_change_password.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="medium-12 columns">
+  <h3>Change Your Password</h3>
+    <form data-abide>
+      <div class="row">
```

Comment:
On these, since we aren't using the grid to adjust the way fields are laid out at different resolutions, we don't need these `row`s and `columns`. We'd be fine with just the `label`s and `input`s inline stacked on top of one abother, but since we're using Abide for field validation (at least now), we need to wrap each form field in a div to contain its error message:

    <form data-abide>
      <div>
        <label>Current Password
          <input type="text" required>
        </label>
        <small class="error">Required</small>
      </div>
      <div>
        <label>New Password
        ...